### DISCUSSION TITLE
How to override this?

### DISCUSSION THREAD

**Discussion by socbaysky** [coderflexx/laravel-turnstile#6] [Q&A]
https://challenges.cloudflare.com/turnstile/v0/siteverify

How to override this link? I want to use proxy because my server run without internet

**Comment by ousid** [comment]
You can use your own __Frontend__ implementation, and skip adding `x-turnstile-widget` component, in your blade layout.

For more info:
https://developers.cloudflare.com/turnstile/get-started/client-side-rendering/
